vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations.cc

```diff
@@ -232,6 +228,14 @@ void Confirmations::Retry() {
   RetryAfterDelay();
 }
 
+void Confirmations::OnDidSendConfirmation(
+    const ConfirmationInfo& confirmation) {
+  BLOG(1, "Successfully sent confirmation with id "
```

Comment:
does this need to be logged at verbosity 1? since it contains creative instance ID which can be linked to user interests

Reply:
Agreed. This is using the same verbosity as other confirmations. Would it be ok to create a follow-up issue where we change all of these together and I can also assess all level 1 verbosities?

Reply:
follow up sgtm

Reply:
https://github.com/brave/brave-browser/issues/16462